G1ANT.Addon.Selenium/Api/SeleniumManager.cs

```diff
@@ -154,28 +154,24 @@ public static void RemoveWrapper(SeleniumWrapper wrapper)
 
         public static void DisposeAllOpenedDrivers()
         {
+            string[] allDriverNames = { "geckodriver", "MicrosoftWebDriver", "IEDriverServer", "chromedriver" };
             List<Process> allDrivers = new List<Process>();
```

Comment:
var allDrivers = new List<Process>();

Reply:
It has to be:
var allDrivers = new List<Process>();

In other case, error is thrown